packages/default/scss/stepper/_variables.scss

```diff
@@ -83,5 +83,5 @@ $kendo-stepper-progressbar-fill-bg: null !default;
 $kendo-stepper-progressbar-fill-text: null !default;
 
 $kendo-stepper-content-transition-property: height !default;
-$kendo-stepper-content-transition-duration: 300ms !default;
+$kendo-stepper-content-transition-duration: var(--kendo-motion-medium-2, 300ms) !default;
```

Comment:
check usage and replace with `kendo-motion(...), better yet search for "transition:" in general